tests/auth-sso-provisioning.unit.test.ts

```
@@ -1,29 +1,9 @@
-  isAdmin: false,
-  plans: [] as any[],
-}
+import { AsyncLocalStorage } from 'node:async_hooks'
+import { describe, expect, it, vi } from 'vitest'
 
-const organizationStore = {
-  organizations: [] as Array<{ gid: string, role: string }>,
-  hasOrganizations: false,
-  fetchOrganizations: vi.fn(async () => {}),
-  dedupFetchOrganizations: vi.fn(async () => {}),
+type MockFetchResponse = {
```

ESLint

[error] Use an `interface` instead of a `type`.

(ts/consistent-type-definitions)